app/views/spree/admin/orders/_form/_adjustments.html.haml

```diff
@@ -3,12 +3,12 @@
     %legend= title
     %table>
       %thead
-        %th= Spree.t('name')
-        %th= Spree.t('amount')
-      %tbody#order-charges.with-border
+        %tr
+          %th= Spree.t('name')
+          %th= Spree.t('amount')
+      %tbody.with-border
         - adjustments.each do |adjustment|
-          - if (adjustment.originator_type != 'Spree::ShippingMethod') && !(adjustment.originator_type == 'Spree::TaxRate' && adjustment.amount == 0)
```

Comment:
I think we can delete admin/orders_spec line 258:
`next if (adjustment.originator_type == 'Spree::TaxRate') && (adjustment.amount == 0)`

Reply:
Done.